gulpfile.js

```diff
@@ -720,7 +723,7 @@ function dist() {
     printConfigHelp('gulp dist --fortesting');
   }
   parseExtensionFlags();
-  return compileCss().then(() => {
+  return compileCss(/* watch */ false, /* opt_compileAll */ true).then(() => {
```

Comment:
Revert.

Reply:
Curious: Why are you setting `watch` to `false` here?

Other parts of the code rely on `watch` being equal to `undefined` in order to determine whether they're in `gulp watch` (once with `watch === true`, once with `watch === false`) or `gulp build | dist` (`watch === undefined`).

(Yes, that's weird design, but it's what we have for now :/)